alerts/kube_apiserver.libsonnet

```diff
@@ -145,6 +145,32 @@ local utils = import 'utils.libsonnet';
               message: 'A client certificate used to authenticate to the apiserver is expiring in less than %s.' % (utils.humanizeSeconds($._config.certExpirationCriticalSeconds)),
             },
           },
+          {
+            alert: 'AggregatedAPIErrors',
+            expr: |||
+              sum by(name, namespace)(increase(aggregator_unavailable_apiservice_count[5m])) > 2
+            ||| % $._config,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'An aggregated API {{ $labels.name }}/{{ $labels.namespace }} have reported errors.',
+              description: 'The number of errors have increased for an aggregated API {{ $labels.name }}/{{ $labels.namespace }} in the five minutes. Note that high values indicate that the availability of the services changes too often.',
+            },
+          },
+          {
+            alert: 'AggregatedAPIDown',
+            expr: |||
+              sum by(name, namespace)(sum_over_time(aggregator_unavailable_apiservice[5m])) > 0
```

Comment:
This should have a for clause of 5 minutes as well and keeping the 5 minute range window. That way if this resolves immediately we don't have a flaky alert.